lib/danger/runner.rb

```diff
@@ -20,13 +21,15 @@ def validate!
     def self.options
       [
         ['--base=[master|dev|stable]', 'A branch/tag/commit to use as the base of the diff'],
-        ['--head=[master|dev|stable]', 'A branch/tag/commit to use as the head']
+        ['--head=[master|dev|stable]', 'A branch/tag/commit to use as the head'],
+        ['--circle-ci-token=[token]', 'A Circle CI API token to be used if needed']
```

Comment:
IMO, kill this option, only allow setting this form an env var, otherwise you'd be leaking it in the build logs